sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -595,32 +595,28 @@ describe("rpc", () => {
                         }
                         callback(undefined, new gempty.Empty());
                     },
-                );
-
-                const engine = createMockEngine((call: any, callback: any) => {
-                    const req: any = call.request;
-                    const severity = req.getSeverity();
-                    const message = req.getMessage();
-                    const urn = req.getUrn();
-                    const streamId = req.getStreamid();
-                    if (severity === engineproto.LogSeverity.ERROR) {
-                        console.log("log error: " + message);
-                    }
```

Comment:
i don't know why we had this to begin with.  but was there a specific reason you removed it?

Reply:
This was here for debugging flaky CI failures - I actually did not intend to remove it, I think it's from a bad merge. I'll add it back in.